index.html

```diff
@@ -3,7 +3,7 @@
 <head>
 <title>Accessible Name and Description Computation 1.2</title>
 <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
-<script src="https://www.w3.org/Tools/respec/respec-w3c-common" class="remove"></script>
+<script src="https://www.w3.org/Tools/respec/respec-w3c-common" class="remove" defer></script>
```

Comment:
If this is needed, it should be in a separate pull request with explanation.

Also, I haven't looked at ReSpec behavior in the last few years, but since it's used to render a lot of the spec contents, I'm not sure you'd want to defer its load. It might trigger the dreaded FOUC: "flash of unstyled/untransformed content."